helm-chart/renku-core/values.yaml

```diff
@@ -16,8 +16,8 @@ cleanupProjectsTTL: 1800
 logLevel: INFO
 
 image:
-  repository: renku/renku-core
-  tag: 'latest'
+  repository: rrrrrok/renku-core
```

Comment:
These two lines should not be committed.

Reply:
thanks - I should actually mark the PR as draft since I haven't had a chance to test the RQ piece yet